### ISSUE TITLE
Redundant calculation of loss during training step

### ISSUE THREAD

**Issue by jhossbach** [zincware/ZnNL#72]
In this code, `value_and_grad` has the ability to both compute the loss and it's gradient at the same time. We can use that and remove the unnecessary second calculation of the loss in `_compute_metrics`.

https://github.com/zincware/ZnRND/blob/36b921aae1580ee4ec64a36219db77e9f3ad27d9/znrnd/models/jax_model.py#L137-L146

**Comment by SamTov** [comment]
@KonstiNik  Did you resolve this at some stage during your restructures?
